crates/bdk/src/descriptor/error.rs

```diff
@@ -64,6 +66,7 @@ impl fmt::Display for Error {
                 f,
                 "The descriptor contains hardened derivation steps on public extended keys"
             ),
+            Self::MultiPath => write!(f, "The descriptor contains multipath keys"),
```

Comment:
```suggestion
            Self::MultiPath => write!(f, "The descriptor contains multipath keys, which are not supported yet"),
```

I think it makes sense to make it explicit that mutipath keys aren't inherently problematic, it's just we don't support it yet.